src/ildasm/dasm.cpp

```diff
@@ -7873,12 +7877,13 @@ BOOL DumpFile()
                     {   // initialized data
                         szptr+=sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s%8.8X = %s (",szTls,fromPtr,KEYWORD("bytearray"));
                         printLine(g_pFile,szString);
-                        sprintf_s(szString,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
+                        szptr = szString;
+                        sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
```

Comment:
That would be true, but then we use `szptr' later, and now realize that the fix should have been:

```szptr += sprintf_s(szptr, SZSTRING_SIZE, "%s                ",g_szAsmCodeIndent);```